network.yaml

```diff
@@ -255,25 +265,29 @@ Resources:
  # Second Availability Zone
   AZBNatGatewayEIP:
     Type: AWS::EC2::EIP
+    Condition: HighlyAvailableTrue
```

Comment:
Thanks for adding this. Let's just name it `HighlyAvailable` since the `true` is implied by the Condition